zubhub_frontend/zubhub/src/views/projects/Projects.jsx

```diff
@@ -57,10 +60,11 @@ function Projects(props) {
   };
 
   const { loading } = state;
-  const { results: projects, previous: prev_page, next: next_page } = props.projects.all_projects;
+  const { count: totalProjects, results: projects, previous: prev_page, next: next_page } = props.projects.all_projects;
   const { hero } = props.projects;
   const staff_picks = props.projects.staff_picks;
   const { t } = props;
+  const noOfPage = Math.ceil(totalProjects / PROJECTS_PAGE_SIZE) ;
```

Comment:
Is `totalProjects` here total projects including drafts (which are not visible)? In that case, pagination calculation will be off

Reply:
> Is `totalProjects` here total projects including drafts (which are not visible)? In that case, pagination calculation will be off

It does not include drafts